prombench/Makefile

```diff
@@ -4,44 +4,58 @@ ifeq ($(AUTH_FILE),)
 AUTH_FILE = /etc/serviceaccount/service-account.json
 endif
 
+ifeq ($(PROVIDER),)
+PROVIDER = gke
+endif
+
 .PHONY: deploy clean
 deploy: nodepool_create resource_apply
 # GCP sometimes takes longer than 30 tries when trying to delete nodepools
 # if k8s resources are not already cleared
 clean: resource_delete nodepool_delete
 
 nodepool_create:
-	$(INFRA_CMD) gke nodepool create -a ${AUTH_FILE} \
-		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
-		-f manifests/prombench/nodepools.yaml
+	${INFRA_CMD} ${PROVIDER} nodepool create -a ${AUTH_FILE} \
+		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} \
+		-v REGION:${REGION} -v NODE_ROLE:${NODE_ROLE} -v ROLE_ARN:${ROLE_ARN} -v SUBNET_IDS:${SUBNET_IDS} -v SEPARATOR:${SEPARATOR} \
+		-v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} 
+		-f manifests/prombench/nodes_${PROVIDER}.yaml
 
 resource_apply:
-	$(INFRA_CMD) gke resource apply -a ${AUTH_FILE} \
-		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} \
+	$(INFRA_CMD) ${PROVIDER} resource apply -a ${AUTH_FILE} \
+		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} \
+		-v REGION:${REGION} \ 
```

Comment:
It think it'll be fine to merge `REGION` and `ZONE` into just `ZONE`, this just seems like documentation and makefile change and change of one line in `eks.go`. 

Any issues you see with combining these two into one?